.github/workflows/build.yml

```diff
@@ -22,7 +22,7 @@ jobs:
 
       - name: Install pmbootstrap from git
         run: |
-          git clone --depth=1 https://git.sr.ht/~postmarketos/pmbootstrap
```

Comment:
git.sr.ht is currently the main pmbootstrap repository. Unfortunately I can't find any news about this, so you'll just have to trust me

Reply:
IIRC it's not anymore, they temporarily moved it to GitLab